src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -21,7 +21,14 @@ namespace Microsoft.AspNet.Mvc.TagHelpers
     /// <remarks>
     /// <see cref="FallbackSrc" /> and <see cref="FallbackTestExpression" /> are required to not be
     /// <c>null</c> or empty to process.
```

Comment:
actually, only the second sentence sounds correct.  I lean toward removing this one unless @DamianEdwards objects.